src/main/java/seedu/address/logic/commands/DurationCommand.java

```diff
@@ -0,0 +1,100 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DURATION;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Duration;
+import seedu.address.model.person.DurationList;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds duration to an existing person in the address book.
+ */
+public class DurationCommand extends Command {
```

Comment:
still a pull request draft and a work in progress, was thinking of changing it session instead of duration

what do you think?

Reply:
I would just follow what's written in the UG since that's what we discussed, but if you think we should change the name then you can bring it up to the group.